nav2_behaviors/plugins/assisted_teleop.cpp

```diff
@@ -77,7 +77,7 @@ void AssistedTeleop::onActionCompletion()
   preempt_teleop_ = false;
 }
 
-ResultStatus AssistedTeleop::onCycleUpdate()
+ResultStatus AssistedTeleop::onCycleUpdate(std::shared_ptr<AssistedTeleopActionResult>)
```

Comment:
Nit: but add the commented out name `/*result*/` like we do elsewhere in the codebase for all eq.